src/content/components/ScrubberViewportIndicator.css

```diff
@@ -0,0 +1,29 @@
+.scrubberViewportIndicator {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  pointer-events: none;
+  z-index: 1;
+}
+
+.scrubberViewportIndicator > div {
```

Comment:
I'd prefer an additional class name for this, so that you don't have an element selector here. E.g. .scrubberViewPortIndicatorSide